pkg/ast/spl/spl.peg

```diff
@@ -2571,6 +2571,47 @@ HeadBlock <- PIPE CMD_HEAD ("limit" EQUAL)? intAsStr:IntegerAsString {
     return queryAgg, nil
 }
 
+
+// Returns *structs.QueryAggregators
+TailBlock <- PIPE CMD_TAIL ("limit" EQUAL)? intAsStr:IntegerAsString {
```

Comment:
Is the `limit EQUAL` required? I don't see it on the SPL Command Tail reference page.

Reply:
addressed in the latest commit